study.md

```diff
@@ -66,15 +69,16 @@ nearest_star = the_sun
 Will modifying `sol` affect `the_sun`? Why or why not?
 
 ```md
-<!-- your answer here -->
+No, because they are their own instances of the object `Star`.
 ```
 
 ## Ruby Object Equality: Part II
 
 Will modifying `the_sun` affect `nearest_star`? Why or why not?
 
 ```md
-<!-- your answer here -->
+No, because `nearest_star` only holds the values of `the_sun`. They aren't
```

Comment:
`the_sun` and `nearest_star` are referencing the same object in memory so modification through one identifier will affect the other.